.github/workflows/protocol-devchain-anvil.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - master
+      - 'release/**'
```

Comment:
Out of curiosity, how do you (personally) test GitHub workflows? I don't know any other way but actually merging the PR, so I was wondering if you have a workaround that helps you make changes with confidence.

Reply:
When wanting to test a workflow, I. usually add my current branch as a workflow trigger, and remove it before merging.